tests/test_simulation.py

```diff
@@ -38,14 +39,23 @@ def testPixelSize(self):
     def testImageShow(self):
         self.sim.images[:].show()
 
+        # Explicitly close all figures before making backend changes.
+        plt.close("all")
```

Comment:
can we just put these in the context/wrapper?

Reply:
yes, looks like I can put it in the `matplotlib_no_gui` context manager. Changing...

Reply:
Awesome! thanks